.gitignore

```diff
@@ -1 +1,3 @@
+.phpunit.result.cache
+composer.lock
```

Comment:
Could you revert this, the lock file is used to ensure this plugin works with the current, lowest, and latest dependencies. Plus it's best practice to commit so who ever picks it up to work on has a set of working dependencies locked and ready to use.

Reply:
Ok